docs/apache-airflow/dag-run.rst

```diff
@@ -111,7 +111,9 @@ in the configuration file. When turned off, the scheduler creates a DAG run only
     """
     from airflow.models.dag import DAG
     from airflow.operators.bash import BashOperator
-    from datetime import datetime, timedelta
+
+    # Note that if you plan to use timezones you should use pendulum
+    from pendulum import datetime, timedelta
```

Comment:
Do we want to also push use of `pendulum.duration()` (the `datetime.timedelta()` equivalent) too?

Reply:
Actually that would require some other code changes I think. Just need to update these docs to still use `datetime.timedelta` since pendulum doesn't have `timedelta`.

Reply:
Ah good point indeed let me try that one and see it.

Reply:
I don't think in this case we need ot to do it. The thing here is to make the "datetime" pendulum, so that whenever you want to add timezone (for example because you realise you need it after the fact), you are not tempted to  use timezone from stdlib - because it causes problems. For example however it would likely be good to separate it visibly at least

Reply:
Well. It is still a mess. Pendulum choosing to implement drop-in replacement for PARTS (?!*&%$) of datetime package  was the next biggest mistake after Python2 encoding choices.

Reply:
Actually I think much better solution will be to add tz to all the examples. I think this is the BEST way to show people best practices.